### ISSUE TITLE
No package 'liblzma' found

### ISSUE THREAD

**Issue by kaigedong** [NixOS/nix#1550]
I am trying to build Nix(Version 1.11.14) for I don't have the root privileges to build the dir /Nix.
But I have the following problem:

```
checking for LIBLZMA... configure: error: Package requirements (liblzma) were not met:

No package 'liblzma' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBLZMA_CFLAGS
and LIBLZMA_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

When I try to install `liblzma` I found the `liblzma` have moved to `lzma` as a part of xz formula.

So, anyone have the solutions?

**Comment by copumpkin** [comment]
Are you building this inside the Nix shell?

**Comment by kaigedong** [comment]
@copumpkin No, I am using a very old server which needs many new dependencies.
 I have found some solutions to solve this problem by installing dependent packages using `linuxbrew` or `conda`.  Thanks a lot.

**Comment by Vaishalinicky** [comment]
No module is available by this name.
 
[root@ansible2 Python-3.9.0]# yum install lzma-devel
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.piconets.webwerks.in
 * epel: ftp.iij.ad.jp
 * extras: mirrors.piconets.webwerks.in
 * updates: mirrors.bkns.vn
No package lzma-devel available.
Error: Nothing to do
[root@ansible2 Python-3.9.0]# yum install liblzma-dev
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.piconets.webwerks.in
 * epel: ftp.iij.ad.jp
 * extras: mirrors.piconets.webwerks.in
 * updates: mirrors.bkns.vn
No package liblzma-dev available.
Error: Nothing to do
[root@ansible2 Python-3.9.0]# yum install liblzma5
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.piconets.webwerks.in
 * epel: nrt.edge.kernel.org
 * extras: mirrors.piconets.webwerks.in
 * updates: mirrors.bkns.vn
No package liblzma5 available.
Error: Nothing to do
[root@ansible2 Python-3.9.0]#

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
